routes/users.py

```diff
@@ -32,6 +32,21 @@ async def register_user(form: models.registration_form):
     # return response in reponse model
     return models.registration_response(user_id=user_id)
 
+@router.delete(
+    "/users/delete",
+    description=docs.registration_desc,
```

Comment:
the doc import name is wrong, should be `docs.delete_user_desc`
(same for summary)